extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -103,36 +100,19 @@ export class Toolbar {
     this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
     const targetId = this.toolbarDOMElement_.getAttribute('target');
```

Comment:
I agree that we could rename target to something else.

However, Most HTML attributes do not contain a dash. perhaps we can change `target` to something like `container`? I'd prefer to have a an attribute name without a dash, but I would be open to `toolba-target` as well